xbus/admin.py

```diff
@@ -24,11 +24,10 @@ class EventAdmin(admin.ModelAdmin):
     list_display = (
         'xref',
         'event_type', 'event_id',
-        'direction', 'state',
         'ctime', 'id',
     )
 
-    list_filter = ('direction', 'state', 'event_type')
+    list_filter = ('xref', 'event_type')
```

Comment:
There will be *a lot* of xrefs, so this field is not suitable to be used in a filter